### PR TITLE
Fix property name case in action sentences

### PR DESCRIPTION
**Pull request by D8H**
It avoids property names to be in lower case in sentences. This issue could have happen with trade marks too.

### CODE REVIEW

**Review by ClementPasteau — APPROVED** [review]
lgtm :)

File: Core/GDCore/String.cpp [hunk 1]
```diff
@@ -403,6 +403,11 @@ String String::LowerCase() const
     return lowerCasedStr;
 }
 
+String String::CapitalizeFirstLetter() const
+{
+  return size() < 1 ? *this : substr(0, 1).UpperCase() + substr(1);
```
Comment (4ian) [inline Core/GDCore/String.cpp:408]: Safety check: can you double check `substr` implementation to verify that calling `substr(1)` on `gd::String("a")` works (return an empty string, does not crash)

**Review by 4ian — APPROVED** [review]

File: Core/GDCore/String.cpp (hunk 1 above)
Comment (ClementPasteau) [reply Core/GDCore/String.cpp:408]: I did check the doc of substr and that's what it's supposed to do, but indeed worth double checking

Comment (D8H) [reply Core/GDCore/String.cpp:408]: With a 1 letter sentence, the character go from lower to upper case without any apparent issue.

**Pull request merged**